run_pacta_data_preparation.R

```diff
@@ -777,6 +777,20 @@ ent_entity_affiliates_last_update <-
   pull(ent_entity_affiliates_last_update) %>%
   unique()
 
+# include FactSet inputs manifest if it is available
+factset_manifest_path <-
+  file.path(
+    config[["factset_data_path"]],
+    paste0(factset_timestamp, "-factset-export-manifest.json")
+  )
+if (file.exists(factset_manifest_path)) {
+  factset_manifest <- jsonlite::fromJSON(factset_manifest_path)
+  factset_manifest <- factset_manifest[c("files", "data_timestamp", "start_time")]
```

Comment:
```suggestion
  factset_manifest <- factset_manifest[c("files", "data_timestamp", "start_time", "metadata_creation_time_date")]
```
Not required, since we're effectively getting a unique timestamp from the `factset_timestamp`, but I think this might be worth bringing along as an additional point.